File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.js
const WORDS = [ "aya", "ye", "woo", "ma"];

function solution(babbling) {
    let count = 0;
    
    for (b of babbling) {
        for (w of WORDS) {
            b = b.replace(w, " ");
        }
        
        if (b.trim() === "") {
            count += 1;
        }
    }
    
    return count;    
}

File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.py
def solution(babbling):
    count = 0
    words = ["aya", "ye", "woo", "ma"]
    
    for b in babbling:
        for w in words:
            b = b.replace(w, " ")
        
        if b.strip() == "":
            count += 1
            
    return count;